src/Microsoft.Diagnostics.Monitoring.HostingStartup/ParameterCapturing/ParameterCapturingService.cs

```diff
@@ -24,7 +24,7 @@ internal sealed class ParameterCapturingService : BackgroundService, IDisposable
 
         public ParameterCapturingService(IServiceProvider services)
         {
-            _logger = services.GetService<ILogger<ParameterCapturingService>>();
+            _logger = services.GetService<ILogger<DotnetMonitor.ParameterCaptureUserCode>>();
```

Comment:
Could we break this down further to `DotnetMonitor.ParameterCapture.UserCode` so that it's easier to wholesale turn on and off the whole parameter capture feature logging?